test/e2e/charts/clickhouse-tls/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: clickhouse-tls
+description: A Helm chart for Kubernetes
```

Comment:
add an actual description, emphasize that this for testing

Reply:
Updated